.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: c
+sudo: required
```

Comment:
why is that required?

Reply:
You're correct, it should work without sudo.